### ISSUE TITLE
Feedback for 0.18.1-alpha.110520200922

### ISSUE THREAD

**Issue by AnthonyLaw** [nemtech/symbol-sdk-typescript-javascript#562]
1. `orderBy` is not working, the transactions is return in ASC sort. It more make sense sort transaction into Desc. 
Example: latest transactions to history transactions

2. `getTransaction()` is not working, pointing the wrong endpoint
current point to `transactions`
it should pont to `transaction`

network: http://api-t172-01.us-west-1.0941-v1.symboldev.network:3000

**Comment by Vektrat** [comment]
- `orderBy` is implicitly set as `id`, maybe you are referring to `order`? why exactly it isn't working?
- the transactions are returned in DESC by default, as far as i know this was not specified in the API, so we chose what was being used before iirc? it may make more sense in this specific case, but pagination is a global setting, for now we are using global defaults as well - ASC which makes more sense in most cases
-2 steven told me to ignore this (though i don't really get it, because we pluralized all the endpoints)
